app/views/absence_requests/_form.html.erb

```diff
@@ -11,6 +11,36 @@
     </div>
   <% end %>
 
+  <div class="field">
+    <%= form.label :creator_id %>
+    <%= form.text_field :creator_id %>
+  </div>
+
+  <div class="field">
+    <%= form.label :start_date %>
+    <%= form.text_field :start_date %>
+  </div>
+
+  <div class="field">
+    <%= form.label :end_date %>
+    <%= form.text_field :end_date %>
+  </div>
+
+  <div class="field">
+    <%= form.label :request_type %>
```

Comment:
I don't think we need request type; this is mostly used internally to cast the request to the right class as it comes out of the db.